Inputs & forms (#286)
adaptyteam/adapty-docs: src/content/docs/guides/flow-builder/builder-inputs-and-forms.mdx

- **State:** Define how the input looks in different situations. Switch between **Default**, **Active**, **Invalid**, and **Disabled** (or add custom states) and apply different visuals to each.
- **Placeholder:** Hint text displayed inside the empty field.
- **Leading and trailing icons:** Add icons inside the input field.
- **Typography:** Text styling for the value displayed in the field.

<ZoomImage id="input-settings-password.webp" width="400px" float="left" alt="Add input" />

Some settings are specific to certain input types:

<ZoomImage id="input-number.webp" width="400px" float="left" alt="Add input" />

| Setting                    | Input types               |
|----------------------------|---------------------------|
| Clear button               | Text, Email               |
| Validate email format      | Email                     |
| Show password icon         | Password                  |
| Edit password requirements | Password                  |
| Number format              | Number                    |
| Date/time format           | Date, Time, Date and time |
| Min and max date           | Date, Date and time       |

### Use input values

> When a user submits an input, you can save the value as a variable using the **On Submit** action. Once saved, the variable is available across the flow — to personalize copy, populate another field, or drive conditional navigation.

Reviewer: 1. Since the name of the action ("On Submit") comes first in the sentence, it makes me think that this section is going to describe an SDK hook. I think it's safe to assume that the reader is new to the concept of Actions in the context of our Flow Builder, so it would be beneficial to elaborate OR just remove the details from this section since in-depth instructions are available below
```suggestion
You can save user input as a variable when they submit a form. Once saved, the variable is available throughout the flow — to personalize, copy, populate another field, or drive conditional navigation. 
```

### Add a quiz

<ZoomImage id="quiz.webp" width="400px" float="left" alt="Add input" />

1. Click **+** at the top left.
2. Select **Quiz**.
3. Choose the quiz type:
   - **Icon/image/emoji options:** A vertical list of selectable options, each with an icon, image, or emoji alongside a text label.
   - **Icon/image/emoji grid:** A grid of selectable options, each with an icon, image, or emoji.
   - **Rating:** A scale for users to express a rating — numeric or star-based.

### Configure quiz actions

Each quiz option is a selectable element. In the **Interactions** panel, each option has its own **Element ID** and a **Group** shared across all options in the quiz. You can add actions that fire when a user taps an option.

To add an action to a quiz option:

1. Select the option.
2. Open the **Interactions** panel on the right.
3. Under **Selectable Triggers**, click **+ Add Action** next to the **On Tap** trigger.
4. Set the action — for example, set a variable or show a navigation button.

Repeat for each option.

<ZoomImage id="quiz-on-tap.webp" width="500px" alt="Add input" />

#### Set up conditional navigation

> For navigation that depends on which option the user selected, add the action to the **navigation button**, not the quiz option itself:

Reviewer: It took me several attempts to understand what this sentence means... Something like:

```suggestion
If  you want user selection to change their flow navigation, modify the conditional logic of the the **navigation button**:
```
